testsuite/libra-fuzzer/src/fuzz_targets.rs

```diff
@@ -28,6 +28,7 @@ static ALL_TARGETS: Lazy<BTreeMap<&'static str, Box<dyn FuzzTargetImpl>>> = Lazy
         Box::new(consensus::ConsensusProposal::default()),
         // Executor
         Box::new(executor::ExecuteAndCommitChunk::default()),
+        Box::new(executor::ExecuteAndCommitBlocks::default()),
```

Comment:
nit: alphabetical by name (i.e., move this new target above ExecuteAndCommitChunk)